_posts/en/newsletters/2025-05-16-newsletter.md

```diff
@@ -34,26 +38,31 @@ Bitcoin's consensus rules._
 - **Proposed BIP for 64-bit arithmetic in Script:** Chris Stewart
   [posted][stewart bippost] a [draft BIP][64bit bip] to the Bitcoin-Dev
   mailing list that proposes upgrading Bitcoin's existing opcodes to
-  operate on 64 numeric values.  This follows his previous research into
-  the subject (see Newsletters [#285][news285 64bit], [#290][news290
+  operate on 64-bit numeric values.  This follows his previous research
+  (see Newsletters [#285][news285 64bit], [#290][news290
   64bit], and [#306][news306 64bit]).  In a change from some of the
-  previous discussion, the new proposal uses numbers in the same
+  earlier discussion, the new proposal uses numbers in the same
   compactSize data format currently used in Bitcoin.  Additional related
   [discussion][stewart inout] occurred on two [threads][stewart
   overflow] on Delving Bitcoin.
 
-<!-- FIXME:harding: ran out of time, will work on some of these topics
-
-- Bram Cohen quines
-- CTV+CSFS BitVM bridges
-- General computation with OP_CAT
-- Wager on OP_CAT activation
-- Post quantum signatures: https://mailing-list.bitcoindevs.xyz/bitcoindev/76c5ec26-5fd5-4746-86ed-89d2c8e28cbc@mattcorallo.com/T/#t
-- Substring search: https://mailing-list.bitcoindevs.xyz/bitcoindev/e1c28747-ccc9-4d14-a93f-77dbe08a0ba9n@googlegroups.com/T/#t
-- Path to general computation on Bitcoin with OP_CAT https://delvingbitcoin.org/t/the-path-to-general-computation-on-bitcoin-with-op-cat/1106
-- Post-sig cross-input scripting: https://delvingbitcoin.org/t/post-signature-cross-input-scripting-using-the-taproot-annex/1520/3
-
--->
+- **Proposed opcodes for enabling recursive covenants through quines:**
+  Bram Cohen [posted][cohen quine] to Delving Bitcoin to suggest a set
+  of simple opcodes that would the creation of recursive
+  [covenants][topic covenants] through self-reproducing scripts
+  (quines).  Cohen describes how the opcodes could be used to create a
```

Comment:
I did not know the term _quine_ before. Could it be linked to https://en.wikipedia.org/wiki/Quine_(computing) ?